python/pyspark/sql/group.py

```diff
@@ -214,15 +214,15 @@ def apply(self, udf):
 
         :param udf: A function object returned by :meth:`pyspark.sql.functions.pandas_udf`
 
-        >>> from pyspark.sql.functions import pandas_udf
+        >>> from pyspark.sql.functions import pandas_udf, PandasUDFType
         >>> df = spark.createDataFrame(
         ...     [(1, 1.0), (1, 2.0), (2, 3.0), (2, 5.0), (2, 10.0)],
         ...     ("id", "v"))
-        >>> @pandas_udf(returnType=df.schema)
+        >>> @pandas_udf("id long, v double", PandasUDFType.GROUP_MAP)
```

Comment:
Just an idea not blocking this PR but I think it might be nicer if we could take a string too for `PandasUDFType.GROUP_MAP` and etc.

Reply:
sounds good, but may bring some trouble when trying to figure out which one is data type and which one is function type...

Reply:
Yup .. it should be something for a followup anyway even it is somehow possible ..